File: core/src/com/symbol/ecs/component/RemoveComponent.java
package com.symbol.ecs.component;

import com.badlogic.ashley.core.Component;
import com.badlogic.gdx.utils.Pool;

public class RemoveComponent implements Component, Pool.Poolable {

    public boolean shouldRemove = false;

    @Override
    public void reset() {
        shouldRemove = false;
    }

}

File: core/src/com/symbol/ecs/component/RemoveComponent.kt
package com.symbol.ecs.component

import com.badlogic.ashley.core.Component
import com.badlogic.gdx.utils.Pool

class RemoveComponent : Component, Pool.Poolable {

    var shouldRemove: Boolean = false

    override fun reset() {
        shouldRemove = false
    }

}